docs/spec_draft.md

```diff
@@ -485,6 +486,71 @@ IEEE-754 specification.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.compare
+
+### Semantics
+
+Performs element-wise comparison of `lhs` and `rhs` tensors according to
+`comparison_direction` and `compare_type`, and produces a `result` tensor. More
+formally, `result[i0, ..., iR-1]` =
+`comparison_direction lhs[i0, ..., iR-1], rhs[i0, ..., iR-1]`.
+
+`comparison_direction` can be one of the following:
+  * `EQ`: `lhs` $=$ `rhs`.
+  * `NE`: `lhs` $\ne$ `rhs`.
+  * `GE`: `lhs` $\ge$ `rhs`.
+  * `GT`: `lhs` $\gt$ `rhs`.
+  * `LE`: `lhs` $\le$ `rhs`.
+  * `LT`: `lhs` $\lt$ `rhs`.
+
+`compare_type` can be one of the following:
+  * `FLOAT`: Standard floating-point comparison of `lhs` and `rhs` as specified
+             in IEEE-754.
+  * `TOTALORDER`: `totalOrder` floating-point comparison of `lhs` and `rhs` as
```

Comment:
Added https://github.com/openxla/stablehlo/issues/223#issuecomment-1312078643
to make sure that the claim is valid.